docusaurus.config.js

```diff
@@ -175,4 +177,4 @@ const config = {
     }),
 };
 
-export default config;
+export default config;
```

Comment:
Your formatter seems to be very aggressive: why you remove newlines at the end of files?